lib/models/contact.dart

```diff
@@ -1,21 +1,23 @@
-import './model.dart';
+import 'package:chirp_nets/models/model.dart';
 
 class Contact implements Model {
   final int id;
   final int conversationId;
+  final int userId;
```

Comment:
what is the difference between id and UserId?

Reply:
contact id and user id, contact id is a unique identifier for the contact object (used for updating/deleting) user id is a way of getting the name from the contact. This has been changed though since contacts are effectively the same as users and this would only be used to get the user

Reply:
so is this a many to many relationship?  like users have contacts and a contact is a link from a user to another user?